packages/components/src/responsive-toolbar-group/style.scss

```diff
@@ -61,7 +61,7 @@
 	width: 100%;
 
 	.responsive-toolbar-group__swipe-list {
-		padding: 0 8px;
+		margin-left: -8px;
```

Comment:
I think we have to customize this style in our design picker. Otherwise, it might look weird if we have already used this component somewhere. For example: the categories shown `/plugins` page

![image](https://user-images.githubusercontent.com/13596067/183050877-511e6161-5ca6-433d-b49c-9a00c6a07b17.png)